rpm/python3-pytz.spec

```diff
@@ -0,0 +1,34 @@
+Summary:       Pytz
+Name:          pytz
+Version:       2022.1
```

Comment:
I would rather set it to `0` from start - it's ignored.

Reply:
Technically speaking, it's not ignored yet as we don't have any tags in the repo ;) But I'll change to 0.